README.rst

```diff
@@ -609,6 +609,39 @@ Cache can be disabled via ``--using-cache`` option or config file:
         ->setUsingCache(false)
     ;
 
+Cache file can be specified via ``--cache-file`` option or config file:
+
+.. code-block:: php
+
+    <?php
+
+    return Symfony\CS\Config\Config::create()
+        ->setCacheFile(__DIR__.'/.php_cs.cache')
+    ;
+
+Speed up builds on Travis
```

Comment:
StyleCI is separate service and it is not needed to give how-to for it in **PHP CS Fixer**.

Maybe, in the future, we could link it (and few more things)